source/includes/ai-engine-guide/_ai-engine-guide.html.md

```diff
@@ -23,7 +23,7 @@ In this section, we outline techniques used by the architect to determine reason
 
 ## Currently Supported Models
 
-At the moment, the Architect chooses from only a small number of solutions. As time goes on, this library of solutions will grow. At the moment we currently support Q-Tables, DQN (Deep Q-Networks) and DDPG (Deep Deterministic Policy Gradients) as our primary reinforcement-learning solutions. In a multi-concept inkling file, the Architect may mix-and-match solutions depending on the complexity of subconcepts.
+At the moment, the Architect chooses from only a small number of solutions. As time goes on, this library of solutions will change. At the moment, we currently support APEX (Distributed Deep Q Network) and PPO (Proximal Policy Optimization). In a multi-concept inkling file, the Architect may mix and match solutions depending on the complexity of subconcepts.
```

Comment:
Apex is slightly different from Distributed Deep Q Network. There is an algorithm called Distributed Deep Q Network, and we do have it implemented, but it is turned off by default. Distributed Deep Q Network only distributes sampling. Apex on the other hand is an asynchronous version of Prioritized Experience replay. In reality what we have is neither Apex nor Distributed Deep Q Network. Ideally it should be called Asynchronous Rainbow DQN or something of that sort, because it includes all the improvements present in Rainbow DQN with some additional improvements (multiple replay buffers) using Asynchronous learning.
Are we sure we want to name this as Distributed Deep Q Network? That might confuse some people, and that is certainly a lesser implementation when compared to what we have.

Reply:
How about "Asynchronous Deep Q Networks"?

Reply:
Yeah, I wasn't sure what term to use here. I borrowed the phrase "distributed deep Q network" from the academic paper that introduced APEX. If there's another description that's better, I'm happy to change it. @AdityaGudimella, do you like Victor's suggestion above, or do you have an alternative suggestion?